common/lib/capa/capa/templates/choicetext.html

```diff
@@ -59,10 +59,12 @@
         <span id="answer_${id}"></span>
     </fieldset>
         <input class= "choicetextvalue" type="hidden"  name="input_${id}{}" id="input_${id}" value="${value|h}" />
-    
+
     <div class="indicator-container">
     % if input_type == 'checkbox' or not element_checked:
-        <span class="status ${status.classname}" id="status_${id}"></span>
+        <span class="status ${status.classname}" id="status_${id}">
+            <span class="sr">${_(status.classname)}</span>
```

Comment:
You can't translate a variable name. It will only work if the text in the variable name happens to have already been scraped somewhere else. The translation has to happen where the actual English text appears as a non-variable, so that the scraper can add it to the dictionary for translations.

There is one thing that you can do to work around this-- you can scrape the text explicitly (but not translate) using ugettext_noop; there are existing examples in the codebase. Otherwise, you need to pass a translated string for the screen-reader text that is different from the classname. I would prefer introducing a different string, as it would be easy to inadvertently change the CSS classname without realizing that it would impact what a screen reader user hears.